### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by jackie8c**
## Overview

This PR creates the Edit Psychiatrist page where psychiatrist information can be entered and headshot can be uploaded. A popup was also created for when a psychiatrist is deleted from the admin side. When a user/psychiatrist is selected, a new nav bar is rendered with the delete button that triggers the popup. 

## Replication

cd frontend
yarn dev
add `/edit_psych` to end of URL to see page for editing psychiatrist profile
add '/filter-users' to end of URL to test popup *testing needs to be done after creating user profiles on firebase

## Changes Made

created `edit_psych.tsx` and `EditPsychiatristProfile.tsx` page, added functionality to dropdowns, added icons to input fields, polished styling

added `upload.svg`, `chevron_down.svg` and `vertical_line.svg` assets

created `psych_profile.tsx` and `ProfessionalProfile.tsx` page and routed to Editing Psychiatrist page, added minor styling

created new 'NavbarTwo.tsx' component to conditionally render when users are selected to be deleted

edited 'Layout.tsx' and 'app.tsx' to support checking for user selection to conditionally render page layout for filterusers page

## Test Coverage

added `/edit_psych` to end of URL
added `/psych_profile` to end of URL

## Next Steps 

Changes on Edit Psychiatrist Profile should be reflected on corresponding profile. 
Also fix styling for popup, edit margins for delete button (show up top left).
Test new navbar using sample users on firebase.

## Related PRs or Issues

#49 #61 

## Screenshots
Screen Shot 2023-10-10 at 5 03 41 PM
Screen Shot 2023-10-10 at 5 04 08 PM

Screen Shot 2023-10-03 at 2 39 41 PM

Screen Shot 2023-09-25 at 11 58 33 AM

<details>

  <summary>Screen Shot Name</summary>

  

</details>

### CODE REVIEW

File: frontend/src/components/EditPsychiatristProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+            <div className="dropdown dropdown-bottom">
+              <div tabIndex={0} className="form-control w-full">
+                <label className="label">
+                  <span className="text-lg">Gender (Required) {chevron_down}</span>
+                </label>
+                <input type="text" placeholder="Select" className="input input-bordered w-full" disabled/>
+              </div>
+              <ul tabIndex={0} className="menu dropdown-content p-2 shadow bg-base-100 rounded-box w-full">
+              {genders.map((e) => <li>{e}</li>)}
```
Comment (DylanTom) [inline frontend/src/components/EditPsychiatristProfile.tsx]: Make sure that the psychiatrist can select their gender. Maybe a single-option checkbox

File: frontend/src/components/EditPsychiatristProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import chevron_down from "@/assets/chevron_down";
+
+const EditPsychiatristProfile = () => {
```
Comment (DylanTom) [inline frontend/src/components/EditPsychiatristProfile.tsx]: Passing in current psychiatrist - use props and pass that in from a parent component.

**Review by DylanTom — COMMENTED** [review]
Great job! Make sure to consider design choices when working on the sprint. Look forward to seeing your work next week.

**Comment by akhil-iyengar-04** [comment]
Updated PR with changes to delete profiles on filter-users page.

**Pull request merged**